lib/util/route_ext.dart

```diff
@@ -393,7 +345,6 @@ extension RouteExt on Route {
           MixpanelProp.title: payload.peerMeta.name,
           MixpanelProp.url: payload.peerMeta.url,
         };
-        break;
```

Comment:
Why we remove `break`?

Reply:
in new version of dart, it auto break in switch-case